api/types.go

```diff
@@ -38,6 +39,19 @@ type networkCreate struct {
 	Options     map[string]interface{} `json:"options"`
 }
 
+// sandboxCreate is the expected body of the "create sandbox" http request message
+type sandboxCreate struct {
+	ID                string                 `json:"id"`
```

Comment:
We can do that too. As of now having SandboxID == ContainerID simplifies operations for clients: Client does not have to store a sandbox id and maintain containerID <=> SandboxID mapping.

But I understand and agree with your point, given Sandbox is a CMN entity, its id should be chosen by CNM.